cws2/migrations/0004_phono_systems.py

```diff
@@ -0,0 +1,266 @@
+# Generated by Django 4.1.7 on 2023-03-31 19:43
+
+import cws2.models.base
+import cws2.models.user
+import cws2.validators
+from django.conf import settings
+import django.contrib.postgres.fields
+import django.core.validators
+from django.db import migrations, models
+import django.db.models.deletion
+import django.utils.timezone
+
+
+class Migration(migrations.Migration):
+    initial = True
```

Comment:
This one's probably my fault as I copy+pasted a migration elsewhere (which I've since fixed) but `initial = True` should only appear on the initial migration, so this doesn't need to be here :)